docs/SETUP.md

```diff
@@ -2,13 +2,27 @@
 
 This is a quick setup guide to walk you through the process of installing and configuring Titania.
 
+## Requirements
+
+1. phpBB 3.1.x, 3.2.x or 3.3.x
```

Comment:
you are sure?

Reply:
Just going by Titania's composer.

https://github.com/phpbb/customisation-db/blob/3.2.x/composer.json#L74